pkg/plugins/config/config.go

```diff
@@ -26,6 +26,13 @@ type PluginManagementCfg struct {
 
 	AngularSupportEnabled  bool
 	HideAngularDeprecation []string
+
+	StackID string
+}
+
+// IsCloud returns true if Grafana is running in Grafana Cloud.
+func (c *PluginManagementCfg) IsCloud() bool {
```

Comment:
Since this is only relevant for something defined in the `pluginsintegration` package, I think it'd be better just using `setting.Cfg` there in this case, as this info is not actually vital for internal plugin management

Reply:
Sounds good since we are in `pluginsintegration` anyways. I will change them :+1:  Thanks!